webpack.test.config.js

```diff
@@ -1,65 +1,12 @@
-var webpack = require( 'webpack' ),
-	path = require( 'path' ),
-	nodeExternals = require( 'webpack-node-externals' );
+const webpack = require( 'webpack' );
+const path = require( 'path' );
+const nodeExternals = require( 'webpack-node-externals' );
+const webpackBaseConfig = require( './webpack.config' );
 
-module.exports = {
```

Comment:
I'm looking at this code side by side with `webpack.config`. There are some significant different. `sassLoader`, `entry`, and several other parameters were not being used. The loader for .jsx files is also different (at least missing eslint). 

Is all this intentional? Have you gone over all of these differences?